### PR TITLE
Proposal

### PR DESCRIPTION
**Pull request by emilkoko**
The two files contain the research proposal and related questions for implementation.

### CODE REVIEW

File: proposal/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Research proposal
+
+I propose the following research project: 
+Given the availability of the ASL database as a global sign language dataset and the need to customize personal datasets for sufficient guessability and high subjective ratings of user-defined gesture sets to produce suitable, context-dependent sentences, how can the design of multimodal input interfaces be optimized to support seamless transitions between different input modes, including tap, motion, and image gestures, and enhance the overall user experience?
+
+For related questions refer to questions.md
```
Comment (tm-mini) [inline proposal/index.md]:
```suggestion
For related questions refer to the [questions](questions.md) file.
```

File: proposal/questions.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# questions
```
Comment (tm-mini) [inline proposal/questions.md]:
```suggestion
# Questions 
```

File: proposal/questions.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# questions 
+
+1. How can I generate my own data?
+2. How can I start a language model locally?
+
+Research proposal questions:
```
Comment (tm-mini) [inline proposal/questions.md]:
```suggestion
## Research Questions
```

File: proposal/questions.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# questions 
+
+1. How can I generate my own data?
```
Comment (tm-mini) [inline proposal/questions.md]:
```suggestion
- How can I generate my own data?
```

File: proposal/questions.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+2. How can I start a language model locally?
```
Comment (tm-mini) [inline proposal/questions.md]:
```suggestion
- How can I start a language model locally?
```

**Review by tm-mini — CHANGES_REQUESTED** [review]

**Review by tm-mini — APPROVED** [review]
From a formatting perspective, I see no issues. I will let @omotolab add comments relating to the actual content itself. For future notes, you can do these things with markdown. 

-  **Make a list, with bold headings**:
```
**Make a list, with bold headings**
```

- **Make headings, from 1 - 6**
# Heading 1
## Heading 2
### Heading 3
#### Heading 4
```
# 1
## 2
### 3
#### 4
```

File: proposal/index.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# Research proposal
+
+I propose the following research project:
```
Comment (omotolab) [inline proposal/index.md]:
```suggestion
```

File: proposal/index.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+# Research proposal
+
+I propose the following research project: 
+Given the availability of the ASL database as a global sign language dataset and the need to customize personal datasets for sufficient guessability and high subjective ratings of user-defined gesture sets to produce suitable, context-dependent sentences, how can the design of multimodal input interfaces be optimized to support seamless transitions between different input modes, including tap, motion, and image gestures, and enhance the overall user experience?
```
Comment (omotolab) [inline proposal/index.md]:
```suggestion
Given the availability of the [ASL database](https://asl-lex.org) as a global sign language dataset, the Neurodiversity Foundation needs a customized personal datasets. We intend to used this for sufficient guessability and high subjective ratings of user-defined gesture sets. We want to produce suitable, context-dependent sentences. Based on this context, we would like to know how we can design a multimodal input interfaces that can be optimized to support seamless transitions between different the following gesture input modes: tap gestures, device motion gestures (via accelerometer and gyroscope data), hand motion gestures (via image/video recognition). The goal is to enhance upon the current standards of keyboard accessibility on devices, by enabling more non-verbal inputs.
```

**Review by omotolab — APPROVED** [review]
I would word some things differently, but at this stage, the goal was to get familiarity with getting reviews before merging into the main repository of this project. We will have time to edit this as we go.

**Pull request merged**